### PR TITLE
Improve documentation TOCs

### PR DESCRIPTION
**Pull request by AltGr**
and some CSS

### CODE REVIEW

File: content/template.xhtml [hunk 1]
```diff
@@ -11,10 +11,10 @@
     <meta name="author" content="OCamlPro" />
     <link rel="icon" type="image/png" href="/ext/img/favicon.png" />
     <link rel="stylesheet" href="/ext/css/bootstrap.css" />
-    <link rel="stylesheet" href="/ext/css/bootstrap-responsive.css" />
     <link rel="stylesheet" href="/ext/css/docs.css" />
     <link rel="stylesheet" href="/ext/js/google-code-prettify/prettify.css" />
     <link rel="stylesheet" href="/ext/css/site.css" />
+    <link rel="stylesheet" href="/ext/css/bootstrap-responsive.css" />
```
Comment (dsheets) [inline content/template.xhtml]: Why was this moved? Now `site.css` can't override.

Comment (AltGr) [inline content/template.xhtml]: From http://getbootstrap.com/2.3.2/components.html#navbar:
> Add .navbar-fixed-top and remember to account for the hidden area underneath it by adding at least 40px padding to the <body>. Be sure to add this after the core Bootstrap CSS and before the optional responsive CSS.

Indeed with the previous order, we would either have no padding on top (hiding the top of pages) on computer screens, or add the padding and have a gap on top of the page on small screens.
